pkg/api/vlabs/validate.go

```diff
@@ -177,13 +177,15 @@ func (a *Properties) Validate() error {
 				return fmt.Errorf("unknown storage type '%s' for agent pool '%s'.  Specify either %s, or %s", agentPoolProfile.StorageProfile, agentPoolProfile.Name, StorageAccount, ManagedDisks)
 			}
 		}
+		/* this switch statement is left to protect newly added orchestrators until they support Managed Disks*/
 		if agentPoolProfile.StorageProfile == ManagedDisks {
 			switch a.OrchestratorProfile.OrchestratorType {
 			case DCOS:
 			case DCOS173:
 			case DCOS184:
 			case DCOS187:
 			case Swarm:
+			case Kubernetes:
```

Comment:
I am not sure i understand what is needed 
if the comment re validated K8S + VMSS = error please check https://github.com/khenidak/acs-engine/blob/f4fee676b2d45a961440e1c19278796d2919525f/pkg/api/vlabs/validate.go#L193 an explicit statement that validates VMSS + K8s 

if the question if K8S + VMSS = Ok then as far as i can tell we can not have that yet. since VMSS unique storage profile per VM is not supported yet. This is needed by Azure volume plugin